src/status_im/core.cljs

```diff
@@ -34,8 +34,13 @@
 
 (def adjust-resize 16)
 
+(defn is-hermes
+  []
+  (boolean (.-HermesInternal js/global)))
```

Comment:
Are those changes necessary?

Reply:
yeah this code helps to find out and prints if `hermes` is enabled or not
I think its useful to keep for future debugging sessions

Reply:
Cool.